live-examples/css-examples/flexbox/flex-shrink.css

```diff
@@ -1,17 +1,42 @@
-.example-container {
+.default-example {
     background-color: #eee;
     border: .75em solid;
     padding: .75em;
-    width: 80%;
+    width: auto;
     max-height: 300px;
     display: flex;
 }
 
-.example-container > div {
+.default-example > div {
     background-color: rgba(0, 0, 255, 0.2);
     border: 3px solid blue;
     margin: 10px;
     flex-grow: 1;
     flex-shrink: 1;
     flex-basis: 300px;
 }
+
+@media screen and (max-width: 760px) {
+
+    .editor-wrapper {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .example-choice-list.large.live,
+    .example-choice-list.live {
+        width: auto;
+    }
+
+    /* Face the triangle down towards the example output */
+    .live .example-choice:before {
+        top: 35%;
+        transform: rotate(90deg);
+    }
+
+    #output {
+        height: auto;
+        width: auto;
+        margin-top: 10px;
+    }
+}
```

Comment:
As mentioned in another PR, I think this is a great idea but, let’s split it out of these changes and come to a consensus of how we can deal with this from a larger perspective and then make the changes based on our conclusion. Thanks!